docs/src/layouts/AppHeader.tsx

```diff
@@ -61,18 +67,54 @@ export default function AppHeader() {
           <HeaderNavBar />
         </Box>
         <Box sx={{ ml: 'auto' }} />
-        <Box sx={{ mr: { xs: 1, md: 2 } }}>
+        <Box sx={{ mr: { xs: 1, md: 1 } }}>
           <DeferredAppSearch />
         </Box>
-        <Box sx={{ display: { md: 'none' }, mr: 1 }}>
-          <HeaderNavDropdown />
-        </Box>
+        <Tooltip title={t('appFrame.github')} enterDelay={300}>
+          <IconButton
+            component="a"
+            color="inherit"
+            href={process.env.SOURCE_CODE_REPO}
```

Comment:
A fun fact, MUI X overrides this value to points to its repository.

```suggestion
            href="https://github.com/mui-org/" 
```

Reply:
Done! Didn't know if this simple approach was the right one 😅

Reply:
I'm not sure if it's the right one, but I'm curious to see what Jun will propose 😁

Reply:
Oops, unresolved then! :)

Reply:
Now we just have to remember to change it when the GitHub org changes. :) (At least that should only ever be once!)

Reply:
@oliviertassinari Sorry, I don't get it. `process.env.SOURCE_CODE_REPO` does not work?

Reply:
It was linking to the /mui-org/material-ui rather than the /mui-org page! That's why the proposed change :)

Reply:
Got it.